pkg/enemy/types.go

```diff
@@ -0,0 +1,76 @@
+package enemy
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/curves"
+	"github.com/genshinsim/gcsim/pkg/core/info"
+	"github.com/genshinsim/gcsim/pkg/shortcut"
+)
+
+var abyssHpMultipliers = map[string]float64{
```

Comment:
rimebiterbathysmalvishap / bolteaterbathysmalvishap should have 1.75 multiplier btw according to https://genshin-impact.fandom.com/wiki/Enemy/Level_Scaling#HP_Multipliers -> update here and in docs

Reply:
is it necessary if the change was one time?
![image](https://github.com/genshinsim/gcsim/assets/35897995/3f5a2132-693a-4b5e-8f8d-8c7056a983ae)

i think to delete them, because they are rarely cases

Reply:
I didn't even notice they were 1 time only, so yeah please delete them.